api/utils.py

```diff
@@ -43,3 +43,26 @@ def get_json(scheme):
         raise RequestError(errors)
 
     return data
+
+
+def cme_form_data(data):
+    if isinstance(data, list):
+        res = {}
+        for element in data:
+            res['note'] = element['note']
+            res['startDate'] = element['startTime']
+            res['info'] = element['cmeAnalyses']
+
+        return res
```

Comment:
Looks like if data will be list of dicts, your function will return only last dict. 
Maybe should use:
res = []
for element in data:
            res.append({ ... })

Reply:
https://github.com/mstoro/nasa_api/pull/3/commits/5ab6719a8dbb63b3ffec202d5e189127bb3357f6